examples/cisco/demo_interface_service_vni.pp

```diff
@@ -15,12 +15,27 @@
 # limitations under the License.
 
 class ciscopuppet::cisco::demo_interface_service_vni {
-  cisco_interface_service_vni { 'Ethernet9/2 344' :
-    encapsulation_profile_vni   => 'vni_500_5000',
-    shutdown                    => true,
-  }
-  cisco_interface_service_vni { 'Ethernet9/2 491' :
-    encapsulation_profile_vni   => 'vni_600_6000',
-    shutdown                    => 'default',
+
+  if platform_get() =~ /n7k/ {
+
+    cisco_encapsulation {"vni_500_5000" :
+      ensure          => present,
+      dot1q_map       => ['500', '5000'],
+    }
+    cisco_interface_service_vni { 'Ethernet9/2 344' :
```

Comment:
Is this the test that needs a special linecard?  If so, should we be checking for that?